nupic/algorithms/CLAClassifier.py

```diff
@@ -213,6 +213,31 @@ def __setstate__(self, state):
     self._version = BitHistory.__VERSION__
 
 
+  def write(self, proto):
+    proto.id = self._id
+
+    statsProto = proto.init("stats", len(self._stats))
+    for (bucketIdx, dutyCycle) in enumerate(self._stats):
+      statsProto[bucketIdx].index = bucketIdx
+      statsProto[bucketIdx].dutyCycle = dutyCycle
+
+    proto.lastTotalUpdate = self._lastTotalUpdate
+    proto.learnIteration = self._learnIteration
+
+
+  def read(self, proto):
```

Comment:
The examples on the wiki page might be out of date for this but I think we are going to implement `read` functions as class methods that construct a new object without initialization with `__new__` and then load everything from the proto. See this example:

https://github.com/numenta/nupic/blob/6c78a70ccbba9e642864b2d9d81714f19b1cb7e1/nupic/encoders/coordinate.py#L195